firefox-ios/Client/Frontend/UserContent/UserScripts/AllFrames/AtDocumentEnd/ContextMenu.js

```diff
@@ -37,5 +42,8 @@ window.__firefox__.includeOnce("ContextMenu", function() {
     if (data.link || data.image) {
       webkit.messageHandlers.contextMenuMessageHandler.postMessage(data);
     }
-  }, true);
+  }
+
+  window.addEventListener("touchstart", sendMessage, true);
+  window.addEventListener("mousedown", sendMessage, true);
```

Comment:
I tried it locally works as expected. I am assuming we handle the long press logic in swift right ? Just out of curiosity, could you point that out please 😄

Reply:
Yes, it's part of ` WKUIDelegate` delegate method: 

`func webView(
        _ webView: WKWebView,
        contextMenuConfigurationForElement elementInfo: WKContextMenuElementInfo,
        completionHandler: @escaping (UIContextMenuConfiguration?) -> Void
    )
`
You can find it in: `BrowserViewController+WebViewDelegates` file.